radio/src/storage/storage_common.cpp

```diff
@@ -100,18 +100,6 @@ void postRadioSettingsLoad()
     g_eeGeneral.internalModule = DEFAULT_INTERNAL_MODULE;
   }
 #endif
-#if defined(STICK_DEAD_ZONE)
-  if (!g_eeGeneral.stickDeadZone) {
-    g_eeGeneral.stickDeadZone = DEFAULT_STICK_DEADZONE;
-  }
-#endif
-#if !defined(DEBUG)
-  // clean up leftovers from a previous DEBUG config
-  for (uint8_t port_nr = 0; port_nr < MAX_AUX_SERIAL; port_nr++) {
-    if (serialGetMode(port_nr) == UART_MODE_DEBUG)
-      serialSetMode(port_nr, UART_MODE_NONE);
-  }
-#endif
```

Comment:
I think you accidentally included this on the rebase

Reply:
Thanks. I've put the code back in.